src/v1/feedback/db.ts

```diff
@@ -2,62 +2,170 @@ import { firebaseDB } from '../auth';
 import { Feedback } from './models/feedback'
 import { DISPLAY_MAP } from '../mapping';
 
+
 export class FeedbackDB {
 
-  // GET feedback of specific location
-  async feedbackListLocation(location: string):
-    Promise<{
-      id: string,
-      data: Feedback[]
-    }[]> {
-    const data = [];
-    if (location && location in DISPLAY_MAP) {
-      const docs = await firebaseDB.collection('feedback').doc(location).collection('feedback').get()
-        .then(doc => {
-          doc.forEach(d => {
-            data.push({ id: d.id, data: d.data() });
-          })
-        })
-        .catch(err => {
-          console.log(err);
-          return err;
-        });
-    }
-    return data;
+  // get hour docs
+  async getHourFeedback(dayRef: FirebaseFirestore.CollectionReference<FirebaseFirestore.DocumentData>, hour: string) {
+    const hourRef = await dayRef.doc(hour).collection('modelPrediction').get();
+    const predicts = hourRef.docs
+      .filter(doc => doc.id != "0")
```

Comment:
Just wondering, why would the feedback when the prediction is 0 not count? Is that like a dummy value added when you initialized the collection?